build-tools/xaprepare/xaprepare/Steps/Step_InstallGradle.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Xamarin.Android.Prepare
+{
+	class Step_InstallGradle : StepWithDownloadProgress, IBuildInventoryItem
```

Comment:
Why do we need `Step_InstallGradle`?  We already have an in-tree `gradlew` in [`build-tools/gradle`](https://github.com/dotnet/android/tree/main/build-tools/gradle).

Reply:
Given that "less code" is generally preferable to "more code", and this code doesn't appear to need to exist at all, I'd go with "remove this xaprepare step".

Regarding "make it easier to update the version", that's a ~1 line change to `build-tools/gradle/gradle/wrapper/gradle-wrapper.properties`, a'la:

https://github.com/dotnet/android/commit/298da1191ecd4078c8ad3eb2606f8bbf738181b7#diff-bd1d26740b9231adf577b4edcf5b9d34d6c845bcfe8a9236f696e9b2ccd8a968R3

```diff
diff --git a/build-tools/gradle/gradle/wrapper/gradle-wrapper.properties b/build-tools/gradle/gradle/wrapper/gradle-wrapper.properties
index 6b3851a8a..fae08049a 100644
--- a/build-tools/gradle/gradle/wrapper/gradle-wrapper.properties
+++ b/build-tools/gradle/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
```

Reply:
I've backed out the xaprepare changes to install gradle, the tests will now use the wrappers in the repo.